packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -123,15 +127,17 @@ const ZoomVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter =>
     };
 
     const recurrence = getRecurrence(event);
+    const userSettings = await getUserSettings();
+
     // Documentation at: https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate
     return {
       topic: event.title,
       type: 2, // Means that this is a scheduled meeting
       start_time: dayjs(event.startTime).utc().format(),
       duration: (new Date(event.endTime).getTime() - new Date(event.startTime).getTime()) / 60000,
-      //schedule_for: "string",   TODO: Used when scheduling the meeting for someone else (needed?)
-      timezone: event.organizer.timeZone,
-      //password: "string",       TODO: Should we use a password? Maybe generate a random one?
+     //schedule_for: "string",   TODO: Used when scheduling the meeting for someone else (needed?)
+     timezone: event.organizer.timeZone,
+     //password: "string",       TODO: Should we use a password? Maybe generate a random one?
```

Comment:
I think we can now do something like 

userSettings. schedule_meeting.default_password_for_scheduled_meetings ?? undefined for setting passwords. As this comes back from the api too